src/Impostor.Api/Innersloth/VentLocation.cs

```diff
@@ -0,0 +1,48 @@
+namespace Impostor.Api.Innersloth
+{
+    public enum VentLocation : uint
+    {
+        // Skeld
+        SkeldAdmin = 0,
+        SkeldRightHallway = 1,
+        SkeldCafeteria = 2,
+        SkeldElectrical = 3,
+        SkeldUpperEngine = 4,
+        SkeldSecurity = 5,
+        SkeldMedbay = 6,
+        SkeldWeapons = 7,
+        SkeldLowerReactor = 8,
+        SkeldLowerEngine = 9,
+        SkeldShields = 10,
+        SkeldUpperReactor = 11,
+        SkeldUpperNavigation = 12,
+        SkeldLowerNavigation = 13,
+
+        // Mira HQ
+        MiraBalcony = 1,
```

Comment:
Why does this one start at 1?

Reply:
> Why does this one start at 1?

Got the ids from here
https://github.com/codyphobe/among-us-protocol/blob/master/07_miscellaneous/04_map_specific_ids_for_vents_and_tasks.md